autoload/skeleton.vim

```diff
@@ -133,6 +133,9 @@ function! skeleton#DoDefaultReplacements(filename)
   endif
 
   call skeleton#Replace('CURSOR', '')
+  " Substitutions move the cursor's location, so calling this last sets the
+  " cursor's positions to wherever @CURSOR@ is " Equivalent to :cursor()
```

Comment:
Looks like there's a typo or two at the end of the comment. How about this instead:

> Move the cursor to the location of the `@CURSOR@` placeholder.

Also please place the comment above the code it describes.